crypto/merkle/tree.go

```diff
@@ -1,22 +1,27 @@
 package merkle
 
 import (
+	"crypto/sha256"
+	"hash"
 	"math/bits"
 )
 
 // HashFromByteSlices computes a Merkle tree where the leaves are the byte slice,
 // in the provided order. It follows RFC-6962.
 func HashFromByteSlices(items [][]byte) []byte {
+	return hashFromByteSlices(sha256.New(), items)
```

Comment:
Note to myself: at some point, we need to search for all occurrences like this and make sure they use a central variable / constant instead (just in case we decide to switch to another hash function instead)